stubs/fpdf2/fpdf/table.pyi

```diff
@@ -0,0 +1,67 @@
+from _typeshed import Incomplete
+from abc import abstractmethod
+from collections.abc import Iterable
+from dataclasses import dataclass
+from io import BytesIO
+from typing_extensions import Literal
+
+from PIL import Image
+
+from .drawing import DeviceGray, DeviceRGB
+from .enums import Align, TableBordersLayout, TableCellFillMode
+from .fonts import FontFace
+from .fpdf import FPDF
+
+DEFAULT_HEADINGS_STYLE: FontFace
+
+class Table:
+    rows: list[Row]
+
+    def __init__(
+        self,
+        fpdf: FPDF,
+        rows: Iterable[Incomplete] = (),
```

Comment:
Docstring: https://github.com/PyFPDF/fpdf2/blob/615db585a53dde9af5874fb0a58f33ee7f84aa2b/fpdf/table.py#L38

```suggestion
        rows: Iterable[Row | str] = (),
```

Reply:
Good catch, you're right!